router/dns_linux.go

```diff
@@ -6,6 +6,47 @@ import (
 )
 
 func (d *DNS) setupResolver(domain string, ip net.IP) error {
+	version, err := linuxRelease()
+
+	if err != nil {
+		return err
+	}
+
+	switch version {
+	case "ubuntu-18.04":
+		if err := installUbuntu1804(domain, ip); err != nil {
+			return err
+		}
+	default:
+		if err := installGenericLinux(domain, ip); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func installUbuntu1804(domain string, ip net.IP) error {
```

Comment:
Probably best to call these `setupResolverUbuntu1804` and `setupResolverGeneric` just for easy reference in the future